autogtp/main.cpp

```diff
@@ -50,26 +51,27 @@ bool fetch_best_network_hash(QTextStream& cerr, QString& nethash) {
         exit(EXIT_FAILURE);
     }
     QString outhash = outlst[0];
-    cerr << "Best network hash: " << outhash << endl;
     QString client_version = outlst[1];
     auto server_expected = client_version.toInt();
-    cerr << "Required client version: " << client_version;
     if (server_expected > AUTOGTP_VERSION) {
-        cerr << endl;
         cerr << "Server requires client version " << server_expected
              << " but we are version " << AUTOGTP_VERSION << endl;
         cerr << "Check https://github.com/gcp/leela-zero for updates." << endl;
         exit(EXIT_FAILURE);
-    } else {
-        cerr << " (OK)" << endl;
+    }
+    if (games_played == 0) {
```

Comment:
I think this is fairly useless output (except the first time) but I can change it to (games_played % 10 == 0) if you want it to appear more frequently but not everytime

Reply:
These can change at any time, so I don't like this. If you think it is too spammy, it should probably remember the original value and report if there's been a change. The current patch looks like the worst of both worlds.

Reply:
You're right, I'll make it remember. 
I think it's a little spammy because the best network hash can also be found in the curl line which is printed with the same frequency